src/coreclr/jit/compiler.cpp

```diff
@@ -1275,8 +1275,12 @@ void DisplayNowayAssertMap()
             fout = _wfopen(strJitMeasureNowayAssertFile, W("a"));
             if (fout == nullptr)
             {
+#if !defined(TARGET_WINDOWS)
```

Comment:
What compiler was complaining about this? These Jit changes should go upstream.

Reply:
`clang` was complaining on Linux, and I didn't think it was appropriate to just disable the warning, since `%ws` expects a `wchar_t*` (and `wchar_t` is 32 bits on Linux), but we're giving it a `char16_t*` in the code below, so I can't see how we'd get anything but garbage at runtime.

But yeah, happy to send this upstream first.

Reply:
Agree this code looks wrong. This is a code path that I don't think anyone has used in years.

Reply:
@pavelsavara do you have any relevant PRs in progress on the upstream repo to add this change to, or should I open a PR for it?

Reply:
> @pavelsavara do you have any relevant PRs in progress on the upstream repo to add this change to, or should I open a PR for it?

Nevermind; this seems easy enough to PR on its own, so I'll do that.

Reply:
https://github.com/dotnet/runtime/pull/104805

Reply:
I don't have coreCLR expertise yet :)